src/ol/style/iconstyle.js

```diff
@@ -426,7 +426,7 @@ ol.style.IconImage_ = function(image, src, size, crossOrigin, imageState) {
    * @type {boolean}
    */
   this.tainting_ = false;
-
+  if (this.imageState_ == ol.style.ImageState.LOADED)this.determineTainting_();
```

Comment:
@spatialillusions Any chance you could fix this coding style issue? If not let me know, then I can do it before merging.